custom/icds_reports/reports/service_delivery_report.py

```diff
@@ -125,7 +125,13 @@ def evaulate_value(row, headers_with_columns):
 
             for header in headers_with_columns:
                 if len(header) == 2:
-                    row_data.append(row[header[1]])
+                    if header[0] in (
+                        'Anganwadi center conducted at least 2 CBEs?',
```

Comment:
its probably safer to match these on the column names, since those are less likely to change than the text